release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2419 @@
+a Customized HAProxy Router].
+
+[[use-a-different-f5-partition]]
+==== Use a Different F5 Partition Other than /Common
+
+With {product-title} 3.6, there is now the added ability to use custom F5
+partitions for properly securing and isolating {product-title} route
+synchronization and configuration.
+
+The default is still `/Common` or global partition if not specified. Also,
+behavior is unchanged if the partition path is not specified.  This new feature
+ensures all the referenced objects are in the same partition, including virtual
+servers (`http` or `https`).
+
+[[ocp-36-support-ipv6-terminated-at-the-router-with-internal-ipv4]]
+==== Support IPv6 Terminated at the Router with Internal IPv4
+
+The router container is able to terminate IPv6 traffic and pass HTTP[S] through
+to the back-end pod.
+
+The IPv6 interfaces on the router must be enabled, with IPv6 addresses listening
+(`::80`, `::443`). The client needs to reach the router node using IPv6.
+IPv4 should be unaffected and continue to work, even if IPv6 is disabled.
+
+[NOTE]
+====
+HAProxy can only terminate IPv6 traffic when the router uses the network stack
+of the host (default). When using the container network stack (`oadm router
+--service-account=router --host-network=false`), there is no global IPv6 address
+for the pod.
+====
+
+[[ocp-36-installation]]
+=== Installation
+
+[[ocp-36-automated-installtion-of-cloudforms]]
+==== Automated installation of CloudForms 4.5 Inside OpenShift (Technology Preview)
+
+The installation of containerized CloudForms inside {product-title} is now part
+of the main installer (currently in xref:ocp-36-technology-preview[Technology
+Preview]). It is now treated like other common components (metrics, logging, and
+so on).
+
+After the {product-title} cluster is provisioned, there is an additional
+playbook you can run to deploy CloudForms into the environment (using the
+`openshift_cfme_install_app flag` in the hosts file).
+
+----
+$ ansible-playbook -v -i <INVENTORY_FILE> playbooks/byo/openshift-cfme/config.yml
+----
+
+Requirements:
+
+[cols="4*", options="header"]
+|===
+|Type
+|Size
+|CPUs
+|Memory
+
+|Masters
+|1+
+|8
+|12 GB
+
+|Nodes
+|2+
+|4
+|8 GB
+
+|PV Storage
+|25 GB
+|N/A
+|N/A
+|===
+
+[NOTE]
+====
+NFS is the only storage option for the Postgres database at this time.
+====
+
+[[ocp-36-installation-of-etcd-docker-daemon-and-ansible-installer-as-system-containers]]
+==== Installation of etcd, Docker Daemon, and Ansible Installer as System Containers (Technology Preview)
+
+RHEL System Containers offer more control over the life cycle of the services
+that do not run inside {product-title} or Kubernetes. Additional system
+containers will be offered over time.
+
+System Containers leverage the OSTree on RHEL or Atomic Host. They are
+controlled by the kernel init system and therefore can be leveraged earlier in
+the boot sequence. This feature is enabled in the installer configuration.
```

Comment:
A little confused about `This feature`, it sounds like `Ansible Installer`, `Docker` and `etcd` system containers would be enabled by the two commands below.

Actually the two commands below is just for `Ansible Installer` system container. `etcd` and `Docker` system containers need some additional parameters individually in installation configuration.

This section seems to talk about system container in overall view. Suggest to move out the two commands below, and add a separate section `Running OpenShift installer as a System Container` under `Configuring System Containers`.